shopno/src/scripts/main.js

```diff
@@ -0,0 +1,9 @@
+import "../styles/style.scss"
+import WorkPopup from "./popUp";
+import ofFormSubmit from "./form";
```

Comment:
Please use more generic. You made typo in name.

Reply:
I fixed it.
```suggestion
    import "../styles/style.scss"
    import ImagePopup from "./imagePopUp";
    import offFormSubmit from "./form";
```